biodatasets/scifact/scifact.py

```diff
@@ -0,0 +1,439 @@
+    """
+
+_BIGBIO_ENTAILMENT_LABELPREDICTION_DESCRIPTION = """\
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    This task is the following: given a claim and several sentences of evidence (that either support or refute the claim), label that sentence with one of {REFUTES, SUPPORTS, NOINFO}. This corresponds to the third task outlined in Section 5 of the paper.
+    """
+
+_DESCRIPTION = {
+    "scifactcorpus_source": _SOURCE_CORPUS_DESCRIPTION,
+    "scifactclaims_source": _SOURCE_CLAIMS_DESCRIPTION,
+    "scifactrationale_bigbio_entailment": _BIGBIO_ENTAILMENT_RATIONALE_DESCRIPTION,
+    "scifactlabelprediction_bigbio_entailment": _BIGBIO_ENTAILMENT_LABELPREDICTION_DESCRIPTION,
+}
+
+_HOMEPAGE = "https://scifact.apps.allenai.org/"
+
+
+_LICENSE = "CC BY-NC 2.0"
+
+_URLS = {
+    _DATASETNAME: "https://scifact.s3-us-west-2.amazonaws.com/release/latest/data.tar.gz",
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "1.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+@dataclass
+class BigBioConfig(datasets.BuilderConfig):
+    """BuilderConfig for BigBio."""
+
+    name: str = None
+    version: str = None
+    description: str = None
+    schema: str = None
+    subset_id: str = None
+
+
+class SciFact(datasets.GeneratorBasedBuilder):
+    """
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="scifactcorpus_source",
+            version=SOURCE_VERSION,
+            description="scifact source schema for the corpus config",
+            schema="source",
+            subset_id="scifact",
+        ),
+        BigBioConfig(
+            name="scifactclaims_source",
+            version=SOURCE_VERSION,
+            description="scifact source schema for the claims config",
+            schema="source",
+            subset_id="scifact",
+        ),
+        BigBioConfig(
+            name="scifactrationale_bigbio_entailment",
+            version=BIGBIO_VERSION,
+            description="scifact BigBio text entailment schema for rationale task",
+            schema="bigbio_entailment",
+            subset_id="scifact",
+        ),
+        BigBioConfig(
+            name="scifactlabelprediction_bigbio_entailment",
+            version=BIGBIO_VERSION,
+            description="scifact BigBio text entailment schema for label prediction task",
+            schema="bigbio_entailment",
+            subset_id="scifact",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "scifact_source"
```

Comment:
might want to choose one of the source subset_id configs here (e.g. scifact_claims_source)

Reply:
Why is it called DEFAULT_CONFIG_NAME when it seems like you want the subset_id to go there?

Reply:
> might want to choose one of the source subset_id configs here (e.g. scifact_claims_source)

Did you make a mistake when you said this? 

My understanding is that `DEFAULT_CONFIG_NAME` should be set to a config name (like `scifact_claims_source`) and not to a subset id (like `scifact_claims`). You said subset_id, but then you put a config name

Reply:
yes, sorry if this was confusing! i should have said "each config has a unique name, you might want to choose a name from one of the configs implementing the source schema here". each config implementing the source schema has a different subset_id and I dropped too many words between brain and keyboard.